ext/standard/tests/math/pow_variation1.phpt

```diff
@@ -62,6 +62,8 @@ $inputs = array(
 /*20*/ "abcxyz",
        'abcxyz',
        $heredoc,
+       '5.5',
+       '2',
 
        // object data
 /*23*/ new classA(),
```

Comment:
The comment likely refers to the iteration and would need to be updated. It's probably best to add the new test cases at the bottom (this also makes the diff easier to review).

Reply:
Thanks. I missed this. Enumeration fixed.